demo-project/src/demo_project/pipelines/reporting/nodes.py

```diff
@@ -90,6 +95,36 @@ def make_price_analysis_image(model_input_table: pd.DataFrame) -> PIL.Image:
     return pil_table.image
 
 
+def create_feature_importance_plot(features: pd.DataFrame):
+    sorted_features = features.sort_values(by="Score", ascending=True)
+    plotly_object = px.bar(
+        sorted_features,
+        x="Score",
+        y="Features",
+        color="Score",
+        color_continuous_scale="RdBu",
+    )
+    return plotly_object
+
+
 def create_matplotlib_chart(companies: pd.DataFrame):
-    plt.plot([1, 2, 3], [4, 5, 6])
+    random_actuals = []
+    random_predicted = []
+    for i in range(0, 11):
+        actual = random.randint(0, 1)
+        predicted = random.randint(0, 1)
+        random_actuals.append(actual)
+        random_predicted.append(predicted)
+
+    data = {"y_Actual": random_actuals, "y_Predicted": random_predicted}
+    plt.style.use("dark_background")
+    df = pd.DataFrame(data, columns=["y_Actual", "y_Predicted"])
+    confusion_matrix = pd.crosstab(
+        df["y_Actual"],
+        df["y_Predicted"],
+        rownames=["Actual"],
+        colnames=["Predicted"],
+        margins=True,
+    )
```

Comment:
I can't leave comment on the confusion matrix, I think it should just have [0,1] there, `all` doesn't fit in there it could be just a 2x2 matrix.

Reply:
ok makes sense, i will convert it to 2x2 then